README.md

```diff
@@ -1,82 +1,86 @@
 # CondeNast/renovate-config
 
-This is a collection of [GitHub-hosted presets](https://docs.renovatebot.com/config-presets/#github-hosted-presets) for Renovate dependency bot.
+This is a collection of [GitHub-hosted presets](https://docs.renovatebot.com/config-presets/#github-hosted-presets) for Renovate dependency bot. 🤖
 
 ## Usage
 
-renovate.json
+In your `renovate.json`:
 
 ```
 {
   "extends": [
-    "config:base",
-    "github>CondeNast/renovate-config:groupMinorUpdates(dependencies)"
-    "github>CondeNast/renovate-config:forceCondenastUpdates",
+    "github>CondeNast/renovate-config",
+    "github>CondeNast/renovate-config:forceCondenastUpdates"
+  ]
+}
```

Comment:
Recommend using the `github>CondeNast/renovate-config` base now instead of `config:base`